src/public/app/dialogs/options/appearance.js

```diff
@@ -181,6 +188,23 @@ const TPL = `
         To content width changes, click on 
         <button class="btn btn-micro reload-frontend-button">reload frontend</button>
     </p>
+    
+    <section>
+        <h4>Version notification</h4>
+        <p>Show a hint in the global menu button when there's a new version available.</p>
+        <div class="form-group row">
+            <div class="col-4">
+                <div>
+                    <input type="checkbox" name="new-version-available" id="new-version-available">
+                    <label for="new-version-available">Inform about new versions</label>
+                </div>
+                <div class="new-minor-version-available">
+                    <input type="checkbox" name="new-minor-version-available" id="new-minor-version-available">
+                    <label for="new-minor-version-available">Also inform about minor versions</label>
+                </div>
+            </div>
+        </div>
+    </section>
```

Comment:
I think this is rather overcomplicated - if the check is not intrusive then there's no need to turn it off. I don't think this needs to be configurable.

Reply:
The second input only makes sense if the first one is checked; So I think it's better UX to disable it to make it clear that the second input (if the first one is disabled) doesn't change anything

Reply:
I think you misunderstand me. What I'm saying is that the feature should be non-intrusive so that there's no point in turning it off. Which means there shouldn't be any configuration at all.

Reply:
I think the feature is non-intrusive but just like [this commenter](https://github.com/zadam/trilium/issues/2319#issuecomment-966206716) just said, there are people who don't want any notification for new updates at all

Reply:
Perhaps, although I'm not sure why this would bother people. Anyway, I would start simple - MVP style. It's very common that features are over-engineered with many options with very little value but high maintenance costs.

Let's make it minimal style with no config, then gather feedback after this is released and then we can improve it iteratively.

Reply:
fixed